.circleci/images/test-image-tamago/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:22.04
+
+RUN apt-get update -y
+RUN apt-get dist-upgrade -y
+
+# create user "builder" with sudo privileges
+ARG GID
+ARG UID
+ARG USER=builder
+RUN groupadd --gid ${GID} $USER
+RUN useradd --uid ${UID} --gid $USER --shell /bin/bash --home-dir /home/$USER --create-home $USER
+RUN apt-get install -y sudo
+RUN echo "builder ALL=(ALL) NOPASSWD: ALL" | tee -a /etc/sudoers
```

Comment:
should this be `$USER` instead of `builder` ?

Reply:
I got kind of confused here, but IIUC builder is the in-container uid and UID and GID are the uid outside the container.

Reply:
it seems to be working ...

Reply:
I tested it interactively and it seems fine.